packages/core-typings/src/Apps.ts

```diff
@@ -45,10 +66,11 @@ export type App = {
 	version: string;
 	price: number;
 	purchaseType: string;
-	pricingPlans: PricingPlan[];
+	pricingPlans: AppPricingPlan[];
 	iconFileContent: string;
 	installed?: boolean;
 	isEnterpriseOnly?: boolean;
+	isPurchased?: string;
 	isSubscribed: boolean;
 	bundledIn: {
```

Comment:
Also missing `bundledAppIds: string[]` if this is an AppOverview. If this is an AppInfo, then this property won't exist.

Reply:
Also, something to be discovered about.